examples/k8s/service/README.md

```diff
@@ -0,0 +1,87 @@
+# Running Hazelcast Go Client on Kubernetes Example
+- This example shows how to run a sample proxy service which uses Hazelcast Go Client connected to the Hazelcast members in the same network and exposes itself to the outside through k8s ingress.
+
+## Table of Contents
+1. [Requirements](#requirements)
+2. [Deployment](#deployment)
+3. [Examples](#examples)
+4. [Deletion of Kubernetes Resources](#deletion-of-kubernetes-resources)
+5. [Running Service Locally](#running-service-locally)
+
+<a name="requirements"></a>
+## Requirements
+- [Minikube](https://minikube.sigs.k8s.io/docs/start/)
+- [Docker](https://www.docker.com/get-started/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+<a name="deployment"></a>
+## Deployment  
+The guide assumes that Docker Engine is ready and running properly. You can run it on other container and virtual machine environments, but it is not tested.
+1. Open a new terminal session.
+2. `minikube start --driver=docker`
+3. `minikube docker-env`
+4. `eval $(minikube -p minikube docker-env)`
+5. Open another new terminal session.
+   1. `minikube addons enable ingress`
+   2. `sudo minikube tunnel`
+6. Go back to previous session and continue on with `docker build -t example/hz-go-service .`
```

Comment:
Could you add a step to clone the Go client repo and cd into the k8s example directory?

Reply:
Added, [67e0826](https://github.com/hazelcast/hazelcast-go-client/pull/743/commits/67e08268ff2729948d21bfd76cac48242408b444).